batch_inference_v2/batch_inference_v2.py

```diff
@@ -32,9 +32,7 @@
 from mlrun.frameworks.auto_mlrun import AutoMLRun
 
 # A union of all supported dataset types:
-DatasetType = Union[
-    mlrun.DataItem, list, dict, pd.DataFrame, pd.Series, np.ndarray, Any
-]
+DatasetType = Union[list, dict, pd.DataFrame, pd.Series, np.ndarray, Any]
```

Comment:
I think it's better to reomve DatasetType - just use "list, dict, pd.DataFrame, pd.Series, np.ndarray" instead.

Reply:
What's the point of a `Union` with `Any`? It's just equivalent to `Any`...

Reply:
`DatasetType` has been removed, seems redundant